### PR TITLE
Ticket/enable deadlineactivity

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/time_based/migrations/0095_auto_20240214_1404.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.20 on 2024-02-14 13:04
+
+from django.db import migrations
+
+
+def enable_deadline_activity(apps, schema_editor):
+    InitiativePlatformSettings = apps.get_model('initiatives', 'InitiativePlatformSettings')
+    settings, _create = InitiativePlatformSettings.objects.get_or_create()
+    settings.activity_types.append('deadlineactivity')
```
Comment (eodolphi) [inline bluebottle/time_based/migrations/0095_auto_20240214_1404.py:9]: This should only happen when period activities are enabled on your platform right?

File: bluebottle/time_based/views/activities.py [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 )
-from bluebottle.time_based.views.mixins import CreatePermissionMixin
-from bluebottle.transitions.views import TransitionList
 
 
 class TimeBasedActivityListView(JsonApiViewMixin, ListCreateAPIView, CreatePermissionMixin):
     permission_classes = (
         ActivityTypePermission,
-        OneOf(ResourcePermission, ActivityOwnerPermission),
+        OneOf(ResourcePermission, ActivityOwnerPermission, IsAdminUser),
```
Comment (eodolphi) [inline bluebottle/time_based/views/activities.py:25]: Doesn't ResourcePermission already return true when you are an admin?

Comment (eodolphi) [reply bluebottle/time_based/views/activities.py:25]: If not, I think it should

**Review by eodolphi — CHANGES_REQUESTED** [review]

File: bluebottle/time_based/migrations/0095_auto_20240214_1404.py (hunk 1 above)
Comment (gannetson) [reply bluebottle/time_based/migrations/0095_auto_20240214_1404.py:9]: I kinda of reckoned that all tenants had this, but I'll make a check just to be sure :ok_hand:

**Comment by gannetson** [comment]
@eodolphi Fixed your remarks

**Review by eodolphi — APPROVED** [review]

**Pull request merged**